commit a7e032a277452366771951e29fd0bf2bd5c029f0
Author: Michael Niedermayer <michael@niedermayer.cc>
Date:   Tue Jul 3 21:37:46 2018 +0200

    avformat/rmdec: Do not pass mime type in rm_read_multi() to ff_rm_read_mdpr_codecdata()
    
    Fixes: use after free()
    Fixes: rmdec-crash-ffe85b4cab1597d1cfea6955705e53f1f5c8a362
    
    Found-by: Paul Ch <paulcher@icloud.com>
    Signed-off-by: Michael Niedermayer <michael@niedermayer.cc>

diff --git a/libavformat/rmdec.c b/libavformat/rmdec.c
index ac61723c66..0216003e88 100644
--- a/libavformat/rmdec.c
+++ b/libavformat/rmdec.c
@@ -522,7 +522,7 @@ static int rm_read_multi(AVFormatContext *s, AVIOContext *pb,
 
         size2 = avio_rb32(pb);
         ret = ff_rm_read_mdpr_codecdata(s, s->pb, st2, st2->priv_data,
-                                        size2, mime);
+                                        size2, NULL);
         if (ret < 0)
             return ret;
     }
